apps/condo/pages/analytics/index.tsx

```diff
@@ -397,13 +412,14 @@ const TicketAnalyticsPageFilter: React.FC<ITicketAnalyticsPageFilterProps> = ({
                 </Col>
                 <Col span={10}>
                     <Form.Item label={AddressTitle} {...FORM_ITEM_STYLE}>
-                        {/* TODO (sitozzz): change to multiple mode */}
-                        <PropertyAddressSearchInput
+                        <GraphQlSearchInput
+                            allowClear
+                            search={searchAddress}
+                            mode={'multiple'}
+                            value={addressList}
+                            onChange={onAddressChange}
+                            maxTagCount={'responsive' as const}
```

Comment:
Why did you use type cast here?

Reply:
So it is written in the antd documentation https://ant.design/components/select/#components-select-demo-responsive